File: C/Iniciante/1041cordenadasponto.c
#include <stdio.h>
int main() {
    double x,y;
    scanf("%lf%lf",&x,&y);
    if(x==0 && y==0){
        //origem
        printf("Origem\n");
    }else if(x==0 && y!=0){
        //eixo x
        printf("Eixo Y\n");
    }else if(x!=0 && y==0) {
        //eixo y
        printf("Eixo X\n");
    }else if(x>0 && y>0){
        //q1
        printf("Q1\n");
    }else if(x<0 && y>0){
        //q2
        printf("Q2\n");
    }else if(x<0 && y<0){
        //q3
        printf("Q3\n");
    }else if(x>0 && y<0){
        //q4
        printf("Q4\n");
    }
    return 0;
}

File: Python/iniciante/1041cordenadasponto.py
linha = input().split(" ")
x, y = linha
x = float(x)
y = float(y)  # type: float
if x == 0 and y == 0:
    # //origem
    print("Origem")
elif x == 0 and y != 0:
    # //eixo x
    print("Eixo Y")
elif x != 0 and y == 0:
    # //eixo y
    print("Eixo X")
elif x > 0 and y > 0:
    # //q1
    print("Q1")
elif x < 0 and y > 0:
    # //q2
    print("Q2")
elif x < 0 and y < 0:
    # //q3
    print("Q3")
elif x > 0 and y < 0:
    # //q4
    print("Q4")
